app/src/main/java/org/schabi/newpipe/util/KoreUtil.java

```diff
@@ -16,6 +18,12 @@ public static boolean isServiceSupportedByKore(final int serviceId) {
                 || serviceId == ServiceList.SoundCloud.getServiceId());
     }
 
+    public static boolean shouldShowPlayWithKodi(final Context context, final StreamInfoItem item) {
```

Comment:
This method can be also used in other places, e.g. in the player, can't it?

Reply:
Do you mean places like
https://github.com/TeamNewPipe/NewPipe/blob/8fe1a76ec6f88556ff8964df09040135c9c51e1e/app/src/main/java/org/schabi/newpipe/player/VideoPlayerImpl.java#L934-L937

Reply:
this seems like a reasonable place to use the new method, but i couldn't find a way to get a `StreamInfoItem` from a `playQueue`. any suggestions?

Reply:
@khimaros Instead of passing in a `StreamInfoItem` you can pass in the service id, since that is the thing you actually need. Otherwise those items should have a common superclass, that would work too, but just passing the id is preferred.

Reply:
that works, thanks!